utils.go

```diff
@@ -122,22 +126,26 @@ func (s *Storage) formatError(op string, err error, path ...string) error {
 	}
 }
 
-func (s *Storage) formatFileObject(v *upyun.FileInfo) (o *types.Object, err error) {
-	o = &types.Object{
-		ID:         v.Name,
-		Name:       s.getRelPath(v.Name),
-		Type:       types.ObjectTypeFile,
-		Size:       v.Size,
-		UpdatedAt:  v.Time,
-		ObjectMeta: info.NewObjectMeta(),
-	}
+func (s *Storage) formatFileObject(v *upyun.FileInfo) (o *typ.Object, err error) {
+	o = s.newObject(false)
+	o.ID = v.Name
+	o.Path = s.getRelPath(v.Name)
+	o.Mode |= typ.ModeRead
+
+	o.SetContentLength(v.Size)
+	o.SetLastModified(v.Time)
+	o.SetServiceMetadata(v.Meta)
```

Comment:
What does `v.Meta` mean here? `x-uss-meta-xyz` should be stored in `UserMetadata`, they belongs to user.

Reply:
`v.Meta` is all the k-v in header with key which has prefix `x-upyun-meta-`

Reply:
https://github.com/upyun/go-sdk/blob/master/upyun/fileinfo.go#L39

Reply:
Yes, they belongs to user intead of service.

Reply:
Also, attch the docs link here.